apps/api/src/routers/admin.py

```diff
@@ -176,23 +171,44 @@ async def submit_review(
         raise HTTPException(status.HTTP_500_INTERNAL_SERVER_ERROR)
 
     if Role.HACKER in applicant_record["roles"]:
-        num_reviewers = applicant_review_processor.get_num_unique_reviewers(
+        unique_reviewers = applicant_review_processor.get_unique_reviewers(
             applicant_record
         )
 
+        update_query: dict[str, object] = {}
+
+        # Only add a review if there are either less than 2 reviewers
+        # or reviewer is one of the reviewers
+        if len(unique_reviewers) < 2 or str(reviewer) in unique_reviewers:
+            update_query.update({"$push": {"application_data.reviews": review}})
+        else:
+            log.error(
+                "%s tried submitting a review, but %s already has two reviewers",
+                reviewer,
+                app,
+            )
+            raise HTTPException(status.HTTP_403_FORBIDDEN)
```

Comment:
Suggestion: since the else clause terminates the control flow, inverting the condition might be easier to follow, i.e.
```python
if len(unique_reviewers) >= 2 and reviewer.uid not in unique_reviewers:
    raise ...

...
```